packages/jellyfish-api-core/src/category/masternode.ts

```diff
@@ -175,6 +175,21 @@ export class Masternode {
     return await this.client.call('setgovheight', [input, activationHeight, utxos], 'number')
   }
 
+  /**
+   * Unset special governance variables
+   *
+   * @param {Record<string, any>} input json object
```

Comment:
From looking at https://github.com/DeFiCh/ain/blob/254bd6292b1ca280b4bbd1273473037cd08c8825/src/masternodes/mn_rpc.cpp#L582

This typing can be:

```suggestion
   * @param {Record<string, number | string | Array<string>>} input json object
```